src/agent/src/rpc.rs

```diff
@@ -519,10 +519,7 @@ impl protocols::agent_ttrpc::AgentService for agentService {
         req: protocols::agent::CreateContainerRequest,
     ) -> ttrpc::Result<Empty> {
         match self.do_create_container(req) {
-            Err(e) => Err(ttrpc::Error::RpcStatus(ttrpc::get_status(
-                ttrpc::Code::INTERNAL,
-                e.to_string(),
-            ))),
+            Err(e) => Err(ttrpc_rpc_error(ttrpc::Code::INTERNAL, e.to_string())),
```

Comment:
LGTM, but please consider these:

- `ttrpc_rpc_error` -> `ttrpc_error` for more shorter
- Since there only 3 types of error code(`INTERNAL`/`INVALID_ARGUMENT `/`UNAVAILABLE `), can we make it more simpler by wrapping `ttrpc_rpc_error(code, msg)` to `ttrpc_internal_error!(msg)`?

Reply:
good idea

Reply:
After thinking about that, I think the ttrpc_internal_error! is not a good name either. Though only 3 error types are used there may be more in the future, we can't define macros for every code. we should not define only one macro for one code because it is inconsistent.